projects/demo/src/modules/components/table/examples/8/index.html

```diff
@@ -0,0 +1,60 @@
+<table
+    tuiTable
+    [columns]="['a', 'b', 'c']"
+>
+    <thead>
+        <tr tuiThGroup>
+            <th
+                *tuiHead="'a'"
+                tuiTh
+                [resizable]="true"
+            >
+                a
+            </th>
+            <th
+                *tuiHead="'b'"
+                tuiTh
+                [resizable]="true"
+            >
+                b
+            </th>
+            <th
+                *tuiHead="'c'"
+                tuiTh
+            >
+                c
+            </th>
+        </tr>
+    </thead>
+    <tbody
+        tuiTbody
+        [data]="[{a: 1, b: 2, c: 3}]"
+    >
+        <tr tuiTr>
+            <td
+                *tuiCell="'a'"
+                tuiTd
+            >
+                a
+            </td>
+            <td
+                *tuiCell="'b'"
+                tuiTd
+            >
+                b
+            </td>
+            <td
+                *tuiCell="'c'"
+                tuiTd
+            >
+                content content
```

Comment:
nit: It would be nice to make a meaningful example. I mean something like "name", "age", "Lorem ipsum dolor sit" instead of  "a", "b", "c", "contentcontentcontent"

Reply:
`contentcontentcontent` is required for overflow line